doc/essentials/EnergyPlusEssentials.tex

```diff
@@ -0,0 +1,2260 @@
+an old stone building, then you also have thermal mass considerations.
+If you use the outside edges there will be too much mass, inside will
+be too little. Again, centerline will split the difference and will
+be very close to the correct amount of thermal mass (possibly losing
+some corner mass).
+
+\subsubsection*{Zones Are Not the Same as Rooms}
+
+A zone sometimes called a ``thermal zone'' is a theoretical construct
+that usually describes a group of rooms that can be treated as a single
+thermal entity. A zone typically includes multiple rooms. Often a
+zone can be seen as a group of rooms that share a single thermostat.
+In the case of many similar rooms with the same thermal and operating
+characteristics, even if many different thermostats are used, they
+may still be grouped into a single zone in EnergyPlus.
+
+\subsubsection*{One Construction Per Surface}
+
+Only one type of construction can be associated with each surface
+so if the top half of a wall is made up of a different construction
+than the bottom half of the wall, the top half and the bottom half
+each need to be represented as separate surfaces.
+
+\subsubsection*{Available Outputs Created by First Simulation}
+
+Since EnergyPlus creates a custom list of possible output variables
+during each simulation, you need to perform a simulation first before
+you can see them. To create the list use the Output:VariableDictionary
+input object and then check the .RDD and .MDD files that are created.
+Select the outputs you want and specify them in the input file.
+
+\subsubsection*{Always Plan Ahead}
+
+Some preliminary steps will facilitate the construction of your input
+file. EnergyPlus requires some information in specified, externally
+available formats; other information may require some lead time to
+obtain. The following checklist should be completed before you start
+to construct your input file.
+\begin{itemize}
+\item Obtain location and design climate information for the city in which
+your building is located. If possible, use one of the weather files
+available for your weather period run.
+\item Obtain sufficient building construction information to allow specification
+of overall building geometry and surface constructions (including
+exterior walls, interior walls, partitions, floors, ceilings, roofs,
+windows, and doors).
+\item Obtain sufficient building use information to allow specification
+of the lighting and other equipment (e.g., electric, gas, etc.) and
+the number of people in each area of the building.
+\item Obtain sufficient building thermostatic control information to allow
+specification of the temperature control strategy for each area of
+the building.
+\item Obtain sufficient HVAC operation information to allow specification
+and scheduling of the fan systems.
+\item Obtain sufficient central plant information to allow specification
+and scheduling of the boilers, chillers and other plant equipment.
+\end{itemize}
```

Comment:
Obtain utility rate info for doing energy calcs?  Line item costing for life cycle analysis?